cloud-sql/postgres/CloudSql.csproj

```diff
@@ -1,13 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Google.Api.Gax" Version="3.7.0" />
     <PackageReference Include="Google.Cloud.Diagnostics.AspNetCore" Version="4.4.0" />
-    <PackageReference Include="Microsoft.AspNetCore.App" />
     <PackageReference Include="MySql.Data" Version="8.0.28" />
```

Comment:
But this one here does not make the deployment fail? Or maybe this one is not needed here as this is the PG sample?

Reply:
>> maybe this one is not needed here as this is the PG sample?

Correct. This is not used. I've updated the CloudSql.csproj file for Postgres to remove this. I also removed the unnecessary package reference to Npgsql in the MySQL CloudSql.csproj file.

Reply:
Perfect thanks!